### PR TITLE
HTBHF-1362 Renamed voucherValueInPence to singleVoucherValueInPence

### PR DESCRIPTION
**Pull request by YetAnotherMatt**

### CODE REVIEW

File: api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/VoucherEntitlementDTO.java [hunk 1]
```diff
@@ -12,21 +13,27 @@
...
+    @ApiModelProperty(notes = "The number of vouchers for pregnancy", example = "1")
     private int vouchersForPregnancy;
 
     @JsonProperty("totalVoucherEntitlement")
+    @ApiModelProperty(notes = "The total number of vouchers the claimant is entitled to", example = "4")
     private int totalVoucherEntitlement;
 
     @JsonProperty("voucherValueInPence")
+    @ApiModelProperty(notes = "The financial value of a single voucher", example = "310")
     private int voucherValueInPence;
```
Comment (YetAnotherMatt) [inline api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/VoucherEntitlementDTO.java]: Note that I have not changed the name used in our API - that remains voucherValueInPence, though I have added notes explaining it.

Comment (mikeSmith08) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/VoucherEntitlementDTO.java]: That seems an odd choice, any reason why? I think calling it `singleVoucherValueInPence` would be helpful and consistent.

Comment (YetAnotherMatt) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/VoucherEntitlementDTO.java]: I've added singleVoucherValueInPence - we'll need to update the front-end then we can remove voucherValueInPence.

Comment (YetAnotherMatt) [reply api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/VoucherEntitlementDTO.java]: Turns out it wasn't used in the front-end, so we've been able to rename it in this PR after all.

**Pull request merged**